manifests/params.pp

```diff
@@ -180,6 +180,14 @@
   $cleanup_timer_interval   = "*-*-* ${fqdn_rand(24)}:${fqdn_rand(60)}:00"
   $dlo_max_age              = 90
 
+  # certificats used for PostgreSQL SSL configuration. Puppet certificates are used
+  $postgresql_ssl_on            = true
+  $postgresql_ssl_folder        = "${puppet_confdir}/ssl"
```

Comment:
After removing the `+=` it fails here because the postgresql user doesn't have permission to read the agent certificates. I think we'll need to copy them to the postgresql user.